R/GenerateRefTables.R

```diff
@@ -287,7 +287,7 @@ GetDetLimitRef <- function() {
   WQXDetLimitRef <- raw.data%>%
     dplyr::mutate(TADA.Limit_Type = dplyr::case_when(
       Name%in%c("Upper Quantitation Limit","Upper Reporting Limit","Upper Calibration Limit") ~ as.character("Over-Detect"),
-      Name%in%c("Drinking Water Maximum","Field Holding Time Limit","Specified in workplan","Statistical Uncertainty","Systematic Uncertainty","Taxonomic Loss Threshold","Water Quality Standard or Criteria") ~ as.character("Other"),
+      Name%in%c("Drinking Water Maximum","Field Holding Time Limit","Specified in workplan","Statistical Uncertainty","Systematic Uncertainty","Taxonomic Loss Threshold","Water Quality Standard or Criteria","Upper 95% Confidence Limit","Lower 95% Confidence Limit") ~ as.character("Other"),
```

Comment:
Can you also add the "Upper 95% Confidence Limit" to the other category in this function, and then run UpdateDetLimitRef() to update the .csv?

Reply:
I think "Upper 95% Confidence Limit" is now in the "other" category, but I'll rerun UpdateDetLimitRef()!